app/services/update_duplicate_matches.rb

```diff
@@ -39,6 +38,7 @@ def create_or_update_fraud_match(match)
       unless existing_fraud_match.candidates.include?(candidate)
         existing_fraud_match.candidates << candidate
         notify_candidate(candidate, existing_fraud_match)
+        block_submission(candidate)
```

Comment:
Shall we group these calls under one method now that they travel together?

Reply:
Yes we shall!

Reply:
Done! Please do let me know if the naming is okay.